src/Essentials/src/Types/LocationExtensions.android.cs

```diff
@@ -24,7 +24,11 @@ internal static Location ToLocation(this AndroidLocation location) =>
 			{
 				Latitude = location.Latitude,
 				Longitude = location.Longitude,
+#if ANDROID34_0_OR_GREATER
+				Altitude = location.HasMslAltitude ? location.MslAltitudeMeters : location.HasAltitude ? location.Altitude : default(double?),
```

Comment:
@jfversluis 
(sorry in advance, if you're not the right person to ping here)

Is it ok if we set MslAltitudeMeters as the default? or would it be better to have another option for it?

The reason why I went for this method is to avoid having to add this for all platforms

Reply:
Hm good question. This is a breaking change because if people rely on `location.Altitude` they will now be forced to use `location.MslAltitudeMeters` and there is no way to opt out.

Can those values be different? Is there a way we can add to make it more optional?

Reply:
According to @cschwarz 's comment on #27554, it should be a default since it offers a more consistent result compared to other platforms. Which checks out, since Apple uses MSL Altitude in [their API too](https://developer.apple.com/documentation/corelocation/cllocation/coordinate)

I would personally go for it. We could also add a `LegacyAltitude` (I'm bad at naming things) property that would represent `Location.Altitude`.